test/unit/specs/components/common/SessionAccounts.spec.js

```diff
@@ -6,32 +6,37 @@ describe(`SessionAccounts`, () => {
   const localVue = createLocalVue()
   localVue.use(Vuelidate)
 
-  let wrapper
+  let wrapper, $store
 
   beforeEach(() => {
+    $store = {
+      state: {
+        accounts: [
+          {
+            name: "accountName",
+            address: "cosmos1ek9cd8ewgxg9w5xllq9um0uf4aaxaruvcw4v9e"
+          }
+        ]
+      }
+    }
     wrapper = shallowMount(SessionAccounts, {
       localVue,
-      stubs: [`router-link`]
+      stubs: [`router-link`],
+      mocks: {
+        $store
+      }
     })
   })
 
   it(`has the expected html structure`, () => {
     expect(wrapper.vm.$el).toMatchSnapshot()
   })
 
-  it(`should close`, () => {
-    const self = {
-      $emit: jest.fn()
-    }
-    SessionAccounts.methods.close.call(self)
-    expect(self.$emit).toHaveBeenCalledWith(`close`)
+  it(`renders the correct account`, () => {
+    expect(wrapper.html()).toContain("<h3>accountName</h3>")
   })
 
-  it("moves to other session pages", () => {
-    const self = {
-      $emit: jest.fn()
-    }
-    SessionAccounts.methods.setState.call(self, "welcome")
-    expect(self.$emit).toHaveBeenCalledWith("route-change", "welcome")
+  it(`should show an account`, () => {
+    expect(wrapper.find(`.card-content`)).toBeDefined()
```

Comment:
This class doesn't exist I believe. Should it be `card.content`? In fact you can put anything in this `find()` and the test still passes.

What you need is:

`expect(wrapper.find(".card.content").exists()).toBe(true)`

p.s. In fact, I don't think you need this test, since the above `accountName` check should be sufficient to prove an account was listed.